clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/model/AmazonReservationReport.groovy

```diff
@@ -61,22 +61,50 @@ class AmazonReservationReport implements ReservationReport {
     OperatingSystemType os
     AtomicInteger totalReserved = new AtomicInteger(0)
     AtomicInteger totalUsed = new AtomicInteger(0)
+    AtomicInteger totalBorrowed = new AtomicInteger(0)
 
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
     Map<String, AccountReservationDetail> accounts = new ConcurrentHashMap<>()
 
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    List<Borrow> borrows = []
+
+    @JsonProperty
+    String id() {
+      return "${region}:${availabilityZoneId()}:${instanceType}:${os.name.toLowerCase()}"
```

Comment:
This new `id` attribute will appear in the existing `v1` and `v2` reports. 

It seems useful to me so I didn't go out of my way (_like I did for `totalBorrowed`_) to make it conditionally appear.

I don't believe existing callers have explicit bindings that will fail on a new attribute, but I'm not confident. This is probably worth a quick follow up.